<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ICoordFormatSetup</class>
 <widget class="QDialog" name="ICoordFormatSetup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>606</width>
    <height>86</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Coordinate Format...</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QRadioButton" name="radioFormat1">
       <property name="text">
        <string>N48° 53.660 E013° 31.113</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioFormat2">
       <property name="text">
        <string>N48.8943° E013.51855°</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioFormat3">
       <property name="text">
        <string>N48° 53' 39.6&quot; E13° 31' 6.78&quot;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ICoordFormatSetup</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ICoordFormatSetup</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
